client_test.go

```diff
@@ -162,7 +162,7 @@ func TestEnableScheduling(t *testing.T) {
 			success:     false,
 			disableAt:   start.Add(time.Hour + time.Second*30),
 			enableAt:    start.Add(time.Hour + time.Second*45),
-			expectRunAt: start.Add(time.Hour + time.Minute),
```

Comment:
I think I didn't understand the purpose of the test when I first looked at it. It is more clear to me after your explanation.

I have reverted the change in test code and add a code to update `next_sched` to the value of `next_run`. See https://github.com/cenkalti/dalga/pull/21/commits/d22e0854345562e8559476d6fc5e74b5569d8800

This was not a problem with constant retry interval but now it has a side effect of resetting exponential backoff to initial value because we lose the original value of `next_sched`. We can leave it that way or if we don't want it we need to put another boolean field to the table that indicates if the job is enabled or not.

Whats your opinion?